README.md

```diff
@@ -41,5 +53,11 @@ Other Notes
 * [Text-Only Mode](Text-Only-Mode.md) (instead of a the graphical interface)
 * [Storing Username/Password](Storing-Username-Password.md) (skipping prompt)
 * [Setting Up PKCS11 CAC Drivers in Ubuntu 16](Setting-Up-PKCS11-CAC-Drivers-in-Ubuntu-16.md)
-* [Profile Support](Profile-Support.md) 
+* [Profile Support](Profile-Support.md) (configurations per remote server)
```

Comment:
Profile support is not per remote server. It is more like per remote services. For example, I have one for all of rite and one for all of je.

Reply:
Fixed